sdk-lib/Cargo.toml

```diff
@@ -12,6 +12,7 @@ edition = "2018"
 [dependencies]
 android_logger = "0.9"
 failure = "0.1"
+hdwallet = "=0.3.0"
```

Comment:
I noticed most of our dependencies are not locked with `=`. Is there a particular reason this one is?

Reply:
The upstream developer merged one of our PRs to bump the `secp256k1` version from 0.19 to 0.21 (a breaking change), and released it as 0.3.1 (which Rust SemVer treats as backwards-compatible). We've migrated to `secp256k1 0.21` in `zcash/librustzcash`, but as of this PR we are still on `secp256k1 0.19` and thus need to pin this temporarily.

Reply:
This has since been fixed; we now depend on `hdwallet >= 0.3.1`.